fc4.rb

```diff
@@ -7,7 +7,7 @@ class Fc4 < Formula
   sha256 "6636e51a5f42ddc99ecbae11e6fa5e846b42f951e75a7720398513037028cb53"
 
   bottle :unneeded
-  depends_on :java => "1.8+"
+  depends_on "adoptopenjdk8"
```

Comment:
Using this works for me.
```suggestion
  depends_on "openjdk@11"
```

Reply:
Yeah. Seems like it may be something wrong with the build rather than the syntax.